backend/cmd/server/main.go

```diff
@@ -136,6 +141,34 @@ func initCacheManager(logger *log.Logger) {
 	cm.Init()
 }
 
+// initAnalytics initializes the analytics service with console adapter and JSON format.
+func initAnalytics(logger *log.Logger) {
+	// Configure analytics to use console adapter with JSON format
+	analyticsCfg := &observability.Config{
+		Enabled: true,
+		Output: observability.OutputConfig{
+			Type:   "console", // Output to stdout for easy viewing
+			Format: "json",    // JSON format as requested
+		},
+		Metrics: observability.MetricsConfig{
+			Enabled: true,
+		},
+		FailureMode: "graceful", // Don't fail if analytics has issues
+	}
+
+	svc, err := observability.InitializeWithConfig(analyticsCfg)
+	if err != nil {
+		logger.Error("Failed to initialize analytics service", log.Error(err))
+		return
+	}
+
+	if svc.IsEnabled() {
+		logger.Info("Analytics service initialized successfully with console adapter and JSON format")
```

Comment:
```suggestion
		logger.Debug("Analytics service initialized successfully with console adapter and JSON format")
```

Do we need this info log?